lib/common.js

```diff
@@ -7,8 +7,14 @@
 //   newEventName - name of the forwarded event (optional)
 //
 const forwardEvent = (from, to, eventName, newEventName = eventName) => {
-  from.on(eventName, (...eventArgs) => {
-    to.emit(newEventName, ...eventArgs);
+  if (from.on) {
+    from.on(eventName, (...eventArgs) => {
+      to.emit(newEventName, ...eventArgs);
```

Comment:
Yes, because it is an internal function and destination is always provided by us, so we can control if it is an EventEmmiter while source is not e.g. W3CWebsocket.
Anyway this PR will be completely rewritten.